OpenRA.Mods.Common/Scripting/Global/ReinforcementsGlobal.cs

```diff
@@ -137,7 +137,7 @@ public Actor[] Reinforce(Player owner, string[] actorTypes, CPos[] entryPath, in
 			}
 			else
 			{
-				var heli = transport.TraitOrDefault<Helicopter>();
+				var heli = transport.TraitOrDefault<Aircraft>();
```

Comment:
This has an implicit dependency on the helicopter logic below.  Can you please add a check on the temporary isPlane flag?